docs/configuration/configuration.md

```diff
@@ -46,6 +46,7 @@ An example [oauth2-proxy.cfg]({{ site.gitweb }}/contrib/oauth2-proxy.cfg.example
 | `--display-htpasswd-form` | bool | display username / password login form if an htpasswd file is provided | true |
 | `--email-domain` | string \| list  | authenticate emails with the specified domain (may be given multiple times). Use `*` to authenticate any email | |
 | `--extra-jwt-issuers` | string | if `--skip-jwt-bearer-tokens` is set, a list of extra JWT `issuer=audience` pairs (where the issuer URL has a `.well-known/openid-configuration` or a `.well-known/jwks.json`) | |
+| `--extra-header` | string \| list | Add additional headers when talking to upstream in raw http header format. eg: `"Foo: Bar"` | |
```

Comment:
I spoke with @NickMeves and we concluded that we'd prefer the option to be `Foo=bar` rather than `foo:bar` as this would be consistent with other options that we  do as key values on the command line